packages/rocketchat-lib/server/functions/sendMessage.js

```diff
@@ -1,10 +1,69 @@
 import _ from 'underscore';
 
+const validateBodyAttachments = (attachments) => {
```

Comment:
As we are now going to be validating the messages and attachments, do we have some place where we document the schema in our documentation?

Reply:
It seens we don't have @graywolf336, only in the [endpoint](https://rocket.chat/docs/developer-guides/rest-api/chat/postmessage/) docs. :(

Reply:
Any recommendations for where to place it?

cc: @MartinSchoeler

Reply:
Hey @MartinSchoeler, any ideas?